packages/ui/src/components/molecules/Send/InputSection.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import SvgGoldQuestionMark from '../../../assets/icons/generated/GoldQuestionMark';
+import {
+  Container,
+  Flex,
+  Typography,
+  Input,
+  SliderCaption,
+  LangDisplay,
+} from '../../atoms';
+import { InfoBox } from '../InfoBox';
+import { Slider } from '../Slider';
+
+interface InputSectionProps {
+  activeButtonId: number;
```

Comment:
Should be renamed to something specifc. Example:

```suggestion
  type: 'slider' | 'input';
```

Reply:
done